src/Mailgun/Mailgun.php

```diff
@@ -166,6 +224,14 @@ public function setApiVersion($apiVersion)
     {
         $this->restClient->setApiVersion($apiVersion);
 
+        foreach ($this->regionClients as $regionClient) {
+            if ($regionClient == null) {
```

Comment:
Changed to strict comparison. I'll keep the Yoda conditions thing in mind, thanks! :)